format-specs/spatial-type-guidance.md

```diff
@@ -0,0 +1,328 @@
+data within an existing type system or format that is otherwise *not* a dedicated
+spatial type. Essentially, if you are storing data in a non-spatial format that you
+intend to be put on a map, and you expect somebody else to interpret it in the same
+way that you did, that data comes from a spatial type implementation.
+
+[^1]: Technically this also applies if you are on the surface of another planet or
+celestial body, and also applies if you are above or below the surface. We'll
+discuss this later.
+
+## If you don't read anything else
+
+The simplest possible spatial type implementation that can interoperate with most existing
+Spatial Type implementations without loosing information is a
+"Geometry" [and/or "Geography"](#geometry-and-geography) type, parameterized with `crs`
+([Coordinate Reference System](#coordinate-reference-systems)) as a string, with values
+encoded as [Well-known binary](#well-known-binary) (if your format has a native way
+to store an array of bytes) or [Well-known text](#well-known-text) (otherwise).
+
+In general, the motivation behind the suggestions in this document are to ensure that
+new Spatial Type implementations:
+
+- **Capture producer intent**: Users of your Spatial Type should not have to discard
+  information when converting spatial data from elsewhere. That means
+  if your Spatial Type intends to interoperate with any other Spatial Type, you need
+  to provide a mechanism to locate a full Coordinate Reference System definition.
+  For example, [GeoPandas](https://geopandas.org) stores a coordinate reference system
+  at the type level of each array of geometries.
+- **Defer to existing standards**: Spatial data can be complicated and there is a lot
+  of prior art to draw from. Deferring to existing standards for well-worn topics like
+  Coordinate Reference System representation and serialized representation of geometry
+  increases the probability that other tools will be able to easily interact with your
+  convention. For example, [GeoParquet](https://geoparquet.org) stores the coordinate
+  reference system as [PROJJSON](#projjson).
+  and stores geometries as [well-known binary](#well-known-binary). It can be tempting
+  to improve on existing spatial standards because many of them have limitations; however,
+  such an improvement whose definition lives in a non-spatial format is more likely to
+  cause confusion and/or misinterpretation than deferring to an existing standard.
+- **Leverage the capabilities of the format**: Every format has a unique set of
+  features and limitations and there is sometimes a tension between deferring to existing
+  standards and ensuring that users of the format can leverage existing tools that work with
+  the format. For example, [GeoJSON](https://geojson.org) encodes geometries as JavaScript
+  objects instead of a text or binary-based format.
+
+For more details and ways to ensure that the type definition is precise and unambiguous,
```

Comment:
This transition sentence seems unnecessary; the next section follows nicely enough without it...